assets/templates/partials/_table-component.html

```diff
@@ -0,0 +1,48 @@
+<table class="dp-c-table">
```

Comment:
How does it define the values for _$bg-color-zebra_, _$font-size-table_ and _$bg-color-hover_ mixing parameters?